api_gateway/views/recomendation.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+import grpc
+from flask import Blueprint, current_app
+from google.protobuf.json_format import MessageToJson
+
+from common.pb2 import recomendations_pb2, recomendations_pb2_grpc
+from views.helpers import returns_json, GRPCException
+
+bp = Blueprint('recomendation', __name__, url_prefix='/recomendations')
+
+RECOMENDATOR_HOST = os.getenv("RECOMENDATOR_HOST", "localhost")
+RECOMENDATOR_PORT = os.getenv("RECOMENDATOR_PORT", "50051")
+
+GRPC_CHANNEL = grpc.insecure_channel(f"{RECOMENDATOR_HOST}:{RECOMENDATOR_PORT}")
+GRPC_STUB = recomendations_pb2_grpc.RecomendationsControllerStub(GRPC_CHANNEL)
+
+@bp.get('/<int:user_id>')
```

Comment:
Let's change this to borrower or borrower_id to be consistent with the model

Reply:
But we are not checking by borrower for BookInstances but for user in the system :/

Reply:
Sure, the model has a foreign key to the user model, but it's called borrower. I just changed it my PR. I don't say it's wrong, but just to be consistent everywhere

Reply:
I dont agree, its true that BookInstance FK is borrower, but in this endpoints we are calling for recomendations to a given user, not for a borrower. It is possible to recommend books to users that have not borrow a single book and then they have no entry in  BookInstance